DataSetExplorer/Program.cs

```diff
@@ -1,159 +1,26 @@
-﻿using DataSetExplorer.DataSetBuilder.Model;
-using DataSetExplorer.DataSetSerializer;
-using System;
-using System.Collections.Generic;
-using System.Collections.Specialized;
-using System.Linq;
-using CodeModel;
-using CodeModel.CaDETModel;
-using CodeModel.CaDETModel.CodeItems;
-using DataSetExplorer.AnnotationConsistencyTests;
-using DataSetExplorer.RepositoryAdapters;
+﻿using DataSetExplorer.RepositoryAdapters;
 
 namespace DataSetExplorer
 {
     class Program
```

Comment:
The Program should be responsible for initializing the ConsoleUI and injecting it with the needed dependencies. You can look at https://github.com/Clean-CaDET/platform/blob/master/SmartTutor/Program.cs for inspiration.

Essentially, the Main should be a single line of code "CreateConsoleUI()", where that private method instantiates the ConsoleUI class and runs it. Later, we will add the Startup for the WebUI and simply invoke a different function (similar to the URL above) in the Program based on input params.

Reply:
I will resolve this in https://github.com/Clean-CaDET/platform/pull/60.